src/guides/v2.4/test/integration/annotations/magento-data-fixture.md

```diff
@@ -82,6 +87,89 @@ Test case that uses the above data fixture: [`dev/tests/integration/testsuite/Ma
 
 [`dev/tests/integration/testsuite/Magento/Cms/Controller/PageTest.php`][] demonstrates an example of the `testCreatePageWithSameModuleName()` test method that uses data from the `cmsPageWithSystemRouteFixture()` data fixture.
 
+### Data Fixture Data Provider
+
+{:.bs-callout-info}
+Data Fixture Data Provider is only applicable to Parameterized Data Fixture and is currently only available for Magento Open Source contributors, and will be released for public with Magento Open Source 2.4.5.
```

Comment:
```suggestion
Data Fixture Data Provider is only applicable to Parameterized Data Fixture and is currently only available for Magento Open Source contributors. It will be released for general use with Magento Open Source 2.4.5.
```